### ISSUE TITLE
Addition of community served open source projects

### ISSUE THREAD

**Issue by gohil-jay** [gohil-jay/Macro-Python-Projects#1]
As hacktoberfest is approaching, this repository can serve as a starting point for various beginners in their open source journey.

If you are one of them, make a contribution and send a PR!

**Comment by Anubhav-1020** [comment]
Hey @gohil-jay ... Can I add small python project to it. It would be really nice if you assign this to me : ).

**Comment by gohil-jay** [comment]
Sure, done. Do make sure you add a **meaningful project** with proper sub-repo structure like the current one.

**Comment by abhishek351** [comment]
yeah can i add tkinter based calculator its a mini project for a python developer

**Comment by abhikalparya** [comment]
Hey @gohil-jay , please assign me this issue.

**Comment by gohil-jay** [comment]
What are you plans for the same? @abhikalparya

**Comment by abhikalparya** [comment]
I can add Dice Roll Simulator and many other mini python projects.
@gohil-jay

**Comment by riya02002** [comment]
sir can I add Quiz Game and some more mini python projects .
@gohil-jay

**Comment by gohil-jay** [comment]
Sure.
